dashboard/controller/classrooms/deleteClassroom.js

```diff
@@ -29,4 +29,4 @@ module.exports = function deleteClassroom(req, res) {
 		});
 		return res.redirect('/dashboard/classrooms');
 	}
-};
+};
```

Comment:
Check your code diff before making a request for review and remove errors like these yourself.